test/util.js

```diff
@@ -0,0 +1,109 @@
+'use strict'
+var sourceMapping = require('./resources/sourceMapping')
+var index = require('../src/index')
+var tape = require('tape')
+var util = require('../src/helpers/util')
+
+tape('Util', function (t) {
+  t.test('lowerbound', function (st) {
+    st.plan(6)
+    var array = [2, 5, 8, 9, 45, 56, 78]
+    var lowerBound = util.findLowerBound(10, array)
+    console.log(array + ' ' + lowerBound)
+    st.ok(lowerBound === 3)
```

Comment:
You do not need to modify it here, but for the future: Please use `st.equal(lowerBound, 3)` - that will give better output in case of errors.